src/app/page.module.css

```diff
@@ -228,3 +228,36 @@
     transform: rotate(0deg);
   }
 }
+
+.header {
+  color: white;
+}
+
+.boxLeft {
+  background-color: purple;
+  width: 300px;
+  height: 250px;
+  margin: 15px;
+  padding: 20px;
+}
+
```

Comment:
Little redundant here!  Could have been a little more efficient with the box styling.

Reply:
Additionally didn't look like the boxes aligned in a row.